README.md

```diff
@@ -1,50 +1,25 @@
-## 18F Front End Guild
+## Front End Guild
 
-18F is promoting team best practices across speciality areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves. [This branch](https://github.com/18F/frontend/tree/research) houses guild research into its membership needs.
+18F promotes team best practices across speciality areas through guilds. These guilds support their members in whatever way deemed most appropriate by those members themselves. [This branch](https://github.com/18F/frontend/tree/research) houses guild research into its membership needs.
 
-This repo is where the 18F Frontend Guild keeps its guide (yep, I went there — guild and guide in one sentence) to best practices and resources for frontend development.
+This repo is where the 18F Front End Guild keeps its guide to best practices and resources for front end development.
 
 #### Quicklinks
-- Published guides: https://frontend.18f.gov/
-- Unpublished guides: [pages](pages/)
-- Drafts: [pulls](https://github.com/18F/frontend/pulls?q=is%3Aopen+label%3Adraft+is%3Apr)
 
-## How to track what we're doing, and how you can be involved!
+- Guide: https://frontend.18f.gov/
 
-We use issues in this repo to track work. They are accessible in a nice visual, prioritized form on [our waffle board](https://waffle.io/18F/frontend).
+## How to track what we're doing, and how you can be involved!
 
-Also, the Frontend Guild Roadmap is in [the wiki](https://github.com/18F/frontend/wiki), and research into what the guild should be doing to support frontend practice at 18F is kept in [this branch](https://github.com/18F/frontend/tree/research).
+We use issues in this repo to track work.
 
 ### How Guild best practices are developed
 
 1. Team members suggest needs in issues, and guild leads identify issues through [research](https://github.com/18F/frontend/tree/research).
```

Comment:
Like commented above, are we still using this branch? Should we update the first step to just suggesting needs in issues?